config.py

```diff
@@ -17,10 +17,12 @@ def getEnvVariable(name):
 POSTGRES_HOST = getEnvVariable("POSTGRES_HOST")
 POSTGRES_PORT = getEnvVariable("POSTGRES_PORT")
```

Comment:
I guess it's better to use something like this and add default value
```python
POSTGRES_PORT = os.environ.get('POSTGRES_PORT', 5432)
```

Reply:
replaced by :
`POSTGRES_PORT = os.environ.get('POSTGRES_PORT', 5432)`
`POSTGRES_PORT = os.environ.get('POSTGRES_DB', "postgres")`